src/component/handlers/edit/DraftEditorEditAndroidHandler.js

```diff
@@ -35,12 +41,30 @@ function onPasteWithSelection(editor, e) {
   });
 }
 
+/**
+ * Note most Android keyboards will only emit the Unidentified key (229)
+ * However it WILL emit Enter (13), we pass this through to support plugins
+ * that trigger on Enter.
+ *
+ * All other interactions are handled with beforeInput.
+ *
+ * @param {DraftEditor} editor
+ * @param {KeyEvent} e
+ */
+function handleKeyDown(editor, e) {
+  const keyCode = e.which;
```

Comment:
⭐️ ⭐️ [which is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which) (as is [keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)). Should we be using [key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead?

Reply:
It mirrors the he rest of Draft's key handling, so for uniformity, I'd like to keep it, at least until we swap it all over.